templates/java-spring/config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Use another name for the file. `config.json` can easily collide with some project template needs. I'd suggest something like `.ag-config.json`.

Reply:
what about `.tp-config.json`?

Reply:
That sounds good. Whatever is not "easy to collide with" 😊